Description: This is causing a failure because makefile is looking for debian/compat, that we don't have anymore.
 Just drop it all, for clearity.
 Upstream has its own (outdated) debian packaging in the repo, so it's not nice to forward this as it is.
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: not-needed
Last-Update: 2021-04-17

--- a/Makefile.am
+++ b/Makefile.am
@@ -37,14 +37,6 @@
   test_run.sh \
   eatmydata.in \
   eatmydata.sh.in \
-  debian/changelog \
-  debian/compat \
-  debian/control \
-  debian/copyright \
-  debian/docs \
-  debian/README.Debian \
-  debian/rules \
-  debian/source \
   libeatmydata.spec
 
 
